stripe/src/main/java/com/stripe/android/view/StripeEditText.kt

```diff
@@ -218,6 +221,24 @@ open class StripeEditText @JvmOverloads constructor(
         fun displayErrorMessage(message: String?)
     }
 
+    override fun onSaveInstanceState(): Parcelable {
+        return bundleOf(
+            STATE_SUPER_STATE to super.onSaveInstanceState(),
+            STATE_ERROR_MESSAGE to errorMessage,
+            STATE_SHOULD_SHOW_ERROR to shouldShowError
+        )
```

Comment:
I like the pattern you used in a recent PR where you create a `data class` to represent the saved state. This helps clarify types.

Reply:
sure, updated